deploy/complete/docker-compose/docker-compose.yml

```diff
@@ -144,76 +149,109 @@ services:
-      - JAVA_OPTS=-Xms32m -Xmx128m -XX:MaxRAM=128m -Dlogging.level.mushop.orders=INFO -Djava.security.egd=file:/dev/urandom -Doracle.jdbc.fanEnabled=false -XX:+UnlockExperimentalVMOptions -XX:+UseZGC 
-
-  payment:
-    image: iad.ocir.io/oracle/ateam/mushop-payment:0.0.1
-    hostname: payment
-    restart: always
-    cap_drop:
-      - all
-    cap_add:
-      - NET_BIND_SERVICE
-    read_only: true
+      - JDBC_URL=jdbc:oracle:thin:system/oracle@oracledborders:1521:xe
+      - JDBC_USERNAME=system
+      - JDBC_PASSWORD=oracle
+      - JDBC_DIALECT=oracle
+      - MICRONAUT_HTTP_SERVICES_USERS_URL=http://user:8080
+      - MICRONAUT_HTTP_SERVICES_CARTS_URL=http://carts:8080
+      - MICRONAUT_HTTP_SERVICES_PAYMENT_URL=http://payment:8080
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "200k"
+        max-file: "5"
 
+  # User service
   user:
-    image: iad.ocir.io/oracle/ateam/mushop-user:1.1.0
-    hostname: user
+    image: iad.ocir.io/cloudnative-devrel/micronaut-showcase/mushop/user:1.0.0-SNAPSHOT
     restart: always
     cap_drop:
       - all
     cap_add:
       - NET_BIND_SERVICE
     read_only: true
+    depends_on:
+      - oracledbuser
     environment:
-      - PORT=80
-      - OADB_USER=mock
-      - OADB_PW=mock
-      - OADB_SERVICE=mock
+      - JDBC_URL=jdbc:oracle:thin:system/oracle@oracledbuser:1521:xe
+      - JDBC_USERNAME=system
+      - JDBC_PASSWORD=oracle
+      - JDBC_DIALECT=oracle
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "200k"
+        max-file: "5"
 
-  nats:
-    image: nats:2.1.2
-    ports:
-      - 4222:4222
-      - 6222:6222
-      - 8222:8222
-    hostname: nats
+  # Payment service
+  payment:
+    image: iad.ocir.io/cloudnative-devrel/micronaut-showcase/mushop/payment:1.0.0-SNAPSHOT
     restart: always
     cap_drop:
       - all
     cap_add:
       - NET_BIND_SERVICE
     read_only: true
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "200k"
+        max-file: "5"
 
-# Fulfillment service
+  # Fulfillment service
   fulfillment:
-    image: iad.ocir.io/oracle/ateam/mushop-fulfillment:1.1.0
-    ports: 
-      - 8087:80
-    hostname: fulfillment
+    image: iad.ocir.io/cloudnative-devrel/micronaut-showcase/mushop/fulfillment:1.0.0-SNAPSHOT
     restart: always
-    cap_drop:
-      - all
-    cap_add:
-      - NET_BIND_SERVICE
     read_only: true
     tmpfs:
       - /tmp:rw,noexec,nosuid
+    depends_on:
+      - nats
     environment:
-      - reschedule:on-node-failure
       - NATS_HOST=nats
       - NATS_PORT=4222
       - ORDERS_NEW=mushop-orders
       - ORDERS_SHIPPED=mushop-shipments
-      - SIMULATION_DELAY=10000
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "200k"
+        max-file: "5"
+
+  nats:
+    image: nats:2.2
+    hostname: nats
+    restart: always
+    cap_drop:
+      - all
+    cap_add:
+      - NET_BIND_SERVICE
+    read_only: true
+
+  oracledbcarts:
```

Comment:
two services are using flyway, so due to internal integrity every service uses it's own database even in docker compose